tests/test_view_errors.py

```diff
@@ -119,7 +119,7 @@ def test_not_found(client):
 
 def test_invalid_body(client):
     response = client.post("/widgets", content_type="text", data="foo")
-    assert_response(response, 400, [{"code": "invalid_body"}])
+    assert_response(response, 400, [{"code": "bad_request"}])
```

Comment:
Latest flask raises `bad_request` on invalid get_json call:
https://flask.palletsprojects.com/en/2.1.x/api/#flask.Request.get_json

Should pass `silent=True` since this framework expects json anyway?